docker-compose.yml

```diff
@@ -18,12 +18,6 @@ services:
       - ./src/main/resources/fields.sql:/docker-entrypoint-initdb.d/fields.sql
```

Comment:
I think `fields` objects are not needed since AI model does not generate its contours